frontend-react/src/shared/Alert/AlertIcon.tsx

```diff
@@ -0,0 +1,54 @@
+import classnames from "classnames";
```

Comment:
I'm thinking that just like the new `<Table />` component, we should just allow the component to be passed in directly for the `<Icon />. IE instead of:

```
<Alert
    headingLevel="h3"
    type="tip"
    icon={{ name: "Lightbulb", size: 3 }}
>
```

we would just pass in the icon component with the size to the component:

```
<Alert
    headingLevel="h3"
    type="tip"
    icon={<Icon.Lightbulb size={3} />}
>
```

Otherwise we'll have to keep maintaining state on this file with what icons we support. Also, keeps the components more stateless.